x-pack/plugins/observability/public/components/alerts_table/common/render_cell_value.tsx

```diff
@@ -100,11 +100,12 @@ export const getRenderCellValue = ({
       case ALERT_SEVERITY:
         return <AlertSeverityBadge severityLevel={value ?? undefined} />;
       case ALERT_EVALUATION_VALUE:
-        const values = getMappedNonEcsValue({
+        const valuesField = getMappedNonEcsValue({
           data,
           fieldName: ALERT_EVALUATION_VALUES,
         });
-        return values ? values : value;
+        const values = getRenderValue(valuesField);
```

Comment:
@maryam-saeidi can you help me understand what `getRenderValue` does? 
I consoled log `valuesField` and `values`, and I saw the same value, but `valuesField` was an array.

Reply:
If the value is a single threshold, both of them are the same, otherwise if the `valuesField` is an array, then it strigifies it. Also, if value is empty, it will return `--` instead.